app/views/ctc/questions/dependents/does_not_qualify_ctc/edit.html.erb

```diff
@@ -1,43 +1,42 @@
-<% @main_question = t("views.ctc.questions.dependents.does_not_qualify_ctc.title", name:  @dependent.first_name.upcase_first) %>
+<% puerto_rico_filing = current_intake.puerto_rico_filing? %>
+<% @main_question = @puerto_rico_filing ? t("views.ctc.questions.dependents.does_not_qualify_ctc.pr.title", name:  @dependent.first_name.upcase_first) : t("views.ctc.questions.dependents.does_not_qualify_ctc.title", name:  @dependent.first_name.upcase_first) %>
```

Comment:
Same thing here - probably doesn't need `@` and not a blocker and feel free to merge with/without this change.